src/app/services/usersApi.ts

```diff
@@ -39,6 +43,10 @@ export const usersApi = api.injectEndpoints({
             query: ({ searchString }) => `/users?search=${searchString}`,
             providesTags: ['Users'],
         }),
+        getUserDetailsById: build.query<UserResponse, UsernameQueryArgs>({
+            query: ({ searchString }) => `/users/userId/${searchString}`,
```

Comment:
Please don't use this route, The route naming is not proper and we might deprecate it real soon